pom.xml

```diff
@@ -512,6 +515,43 @@
         </configuration>
       </plugin>
 
+      <!-- https://mvnrepository.com/artifact/com.diffplug.spotless/spotless-maven-plugin -->
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <version>${spotless.version}</version>
+        <configuration>
+          <java>
+            <excludes>
+              <exclude>**/generated-sources/**</exclude>
+            </excludes>
+            <toggleOffOn/>
+            <removeUnusedImports/>
+            <importOrder/>
+            <trimTrailingWhitespace/>
+            <palantirJavaFormat/>
+            <endWithNewline/>
+            <formatAnnotations/>
+            <indent>
+              <tabs>true</tabs>
+              <spacesPerTab>4</spacesPerTab>
+            </indent>
```

Comment:
Does it mean we allow tabs for indention? (Personally, I don't like tabs because it might be rendered differently on different systems.)

Reply:
No, tabs are _not_ allowed by this configuration. This block is a way to make the linter work as it should with two space indents as used by Parquet, instead of four.

Reply:
It is a bit hacky, but it also allows contributors to run the `spotless:apply` goal in case they are for whatever reason using tabs in their development environment.